app/models/selected_assistance_expert.rb

```diff
@@ -10,7 +10,14 @@ class SelectedAssistanceExpert < ApplicationRecord
 
   scope :not_viewed, (-> { where(expert_viewed_page_at: nil) })
   scope :of_expert, (->(expert) { joins(:assistance_expert).where(assistances_experts: { expert: expert }) })
-  scope :of_diagnoses, (proc do |diagnoses|
+  scope :of_diagnoses, (lambda do |diagnoses|
     joins(diagnosed_need: :diagnosis).where(diagnosed_needs: { diagnosis: diagnoses })
   end)
+  scope :with_status, (->(status) { where(status: status) })
+  scope :updated_before_five_days_ago, (-> { where('updated_at < ?', 5.days.ago) })
```

Comment:
Why five days ? why not a full week ? 
And wouldn't be clearer to write `updated_more_than_five_days_ago` ?

Reply:
Five days = not too short, not spamming the expert ; but not too long, not waiting two weeks before contacting it. Well that's a try.
Ok for `updated_more_than_five_days_ago`